2-ui/1-document/10-size-and-scroll-window/article.md

```diff
@@ -56,13 +56,13 @@ let scrollHeight = Math.max(
 alert('Full document height, with scrolled out part: ' + scrollHeight);
 ```
 
-Why so? Better don't ask. These inconsistencies come from ancient times, not a "smart" logic.
+Why so? Well, the gist of it is these inconsistencies come from ancient times.
```

Comment:
Again, would leave as is. Humanizing. Even if 'not a "smart" logic' is a bit unusual, grammatically.

Reply:
It's too awkward I think. I found it to be inconsistent with the rest of the writing style in the article, especially when read top-down. Maybe we can rephrase it to be more idiomatic?

```suggestion
Why, you might ask? These inconsistencies come from ancient times, not bad design.
```

Reply:
Either way. I still prefer the original, though.

Reply:
```suggestion
Why so? Better don't ask. These inconsistencies come from ancient times, not a "smart" logic.
```